### PR TITLE
fix #4574 - CQ truncate start time instead of round and RecomputePreviousN >= 1

### LINKED ISSUE #4574: CQ current interval start time calculated wrong
When a CQ is run, the start time of the current interval is:
```
interval, err := cq.q.GroupByInterval()
startTime := now.Round(interval)
```
Half way through an interval, the current interval gets rounded up (into the future) to the next interval.  It should use `now.Truncate(interval)` instead.

### PR DESCRIPTION
**Pull request by dgnorton**

### CODE REVIEW

File: services/continuous_querier/service.go [hunk 1]
```diff
@@ -225,9 +229,6 @@ func (s *Service) runContinuousQueries(req *RunRequest) {
 
 // ExecuteContinuousQuery executes a single CQ.
 func (s *Service) ExecuteContinuousQuery(dbi *meta.DatabaseInfo, cqi *meta.ContinuousQueryInfo, now time.Time) error {
-	// TODO: re-enable stats
```
Comment (otoolep) [inline services/continuous_querier/service.go]: Why this change?

Comment (otoolep) [inline services/continuous_querier/service.go]: Though I think it would be better named `cqExecuted`.

**Comment by dgnorton** [comment]
No longer relevant after @jsternberg 's CQ refactoring.
